x-wing/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "x-wing"
+description = """
+Pure Rust implementation of the X-Wing Key-Encapsulation Mechanism
+"""
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.81"
+license = "Apache-2.0 OR MIT"
+readme = "README.md"
+homepage = "https://github.com/RustCrypto/KEMs/tree/master/ml-kem"
+repository = "https://github.com/RustCrypto/KEMs"
+categories = ["cryptography", "no-std"]
+keywords = ["crypto", "x-wing", "xwing", "kem", "post-quantum"]
+exclude = ["src/test-vectors.json"]
+
+
+[features]
+getrandom = ["rand_core/getrandom"]
+
+[lints.clippy]
+pedantic = "warn"                                 # Be pedantic by default
+similar_names = { level = "allow", priority = 1 } # So we can use the names as in the RFC
+
+[lints.rust]
+missing_docs = "deny" # Require all public interfaces to be documented
+
+[dependencies]
+rand_core = { version = "0.6", default-features = false }
+x25519-dalek = { version = "2.0", features = [
```

Comment:
```suggestion
x25519-dalek = { version = "2.0", default-features = false, features = [
```

Reply:
Minimized dependencies, and no-std build should work now. Got confused in the build info as it mentions that the build of ml-kem is broken. Maybe I copied some github build config wrong?